model/version.go

```diff
@@ -41,7 +41,7 @@ type Version struct {
 	GitTags           []GitTag `bson:"git_tags,omitempty" json:"git_tags,omitempty"`
 	TriggeredByGitTag GitTag   `bson:"triggered_by_git_tag,omitempty" json:"triggered_by_git_tag,omitempty"`
 
-	// Parameters stores both user-defined parameters and default parameters
+	// Parameters stores user-defined parameters
```

Comment:
Sorry for the unrelated comment change, I realized it was confusing when looking at something else